There's a parenthesis too much too it seems:

```
/home/runner/work/IfcOpenShell/IfcOpenShell/src/ifcgeom/kernels/opencascade/IfcGeomTree.h:1369:122: error: expected ‘,’ or ‘;’ before ‘)’ token
 1369 |                                 TopoDS_Shape compound(std::move((ifcopenshell::geometry::OpenCascadeShape*)shp)->shape()));
```

--- a/src/ifcgeom/kernels/opencascade/IfcGeomTree.h
+++ b/src/ifcgeom/kernels/opencascade/IfcGeomTree.h
@@ -1365,9 +1365,9 @@ namespace IfcGeom {
 			}
 
 			std::vector<T> select(const IfcGeom::BRepElement* elem, bool completely_within = false, double extend = -1.e-5) const {
-                auto shp = (ifcopenshell::geometry::OpenCascadeShape*)elem->geometry().as_compound();
-				TopoDS_Shape compound(std::move((ifcopenshell::geometry::OpenCascadeShape*)shp)->shape()));
-                delete shp;
+				auto shp = (ifcopenshell::geometry::OpenCascadeShape*)elem->geometry().as_compound();
+				TopoDS_Shape compound(std::move(((ifcopenshell::geometry::OpenCascadeShape*)shp)->shape()));
+				delete shp;
 
 				const auto& m = elem->transformation().data()->ccomponents();
 				gp_Trsf tr;

